floor_generator/lib/processor/queryable_processor.dart

```diff
@@ -21,33 +23,81 @@ abstract class QueryableProcessor<T extends Queryable> extends Processor<T> {
       : assert(classElement != null),
         _queryableProcessorError = QueryableProcessorError(classElement);
 
+  List<FieldElement> get _allFields => [
+        ...classElement.fields,
+        ...classElement.allSupertypes.expand((type) => type.element.fields),
+      ];
```

Comment:
💡This can be moved to the constructor so it's only being initialized once instead of on every access.